eng/common/scripts/Package-Properties.ps1

```diff
@@ -108,13 +108,13 @@ function Get-PkgProperties
 # Takes ServiceName and Repo Root Directory
 # Returns important properties for each package in the specified service, or entire repo if the serviceName is not specified
 # Returns an Table of service key to array values of PS Object with properties @ { pkgName, pkgVersion, pkgDirectoryPath, pkgReadMePath, pkgChangeLogPath }
-function Get-AllPkgProperties ([string]$ServiceDirectory = $null)
+function Get-AllPkgProperties ([string]$ServiceDirectory = $null, [string]$outDirectory)
```

Comment:
This is a common function that I'm not sure is worth passing in this new parameter everywhere for. Instead of doing this lets just add the data to the PackageInfo as an added property and then it will get written to the package info json files and we can either read it from there or get it from the properties and write the txt file outside of this function.